### DISCUSSION TITLE
[Frontend] React + Statium

### DISCUSSION THREAD

**Discussion by nohuhu** [realworld-apps/realworld#860] [Added implementations]
Complete implementation: https://github.com/statiumjs/realworld-example
Live demo: https://react-statium-realworld.netlify.app

**Comment by nohuhu** [comment]
@geromegrignon Is there a way to move this to Added Implementations category? I didn't see an option to post there directly, and this implementation is in fact complete.

**Reply by geromegrignon** [reply]
Hello, please submit your implementation here: https://codebase.show/projects/realworld/implementations/submit

**Reply by nohuhu** [reply]
I did, got it declined:

    Statium is not popular enough. A minimum of 300 GitHub stars is required..

**Reply by nohuhu** [reply]
As a side note, it would have been nice to know about this limitation upfront. Spending significant time on a full implementation and then finding out that it doesn't get past face control at the Cool Kids Club is kinda meh experience.

**Reply by geromegrignon** [reply]
Can you submit it once more, please?
It was rejected by another maintainer, i'll reach him out.

**Reply by nohuhu** [reply]
Gotcha, submitted again. Thanks Gerome!

**Reply by geromegrignon** [reply]
> As a side note, it would have been nice to know about this limitation upfront. Spending significant time on a full implementation and then finding out that it doesn't get past face control at the Cool Kids Club is kinda meh experience.

That's why I want to accept yours 🚀 
I really appreciate time spent on it.

**Comment by geromegrignon** [comment]
Implementation added on [CodebaseShow](https://codebase.show/projects/realworld?category=frontend).

**Comment by nohuhu** [comment]
Thanks Gerome!
